webapp/src/components/jira_ticket_tooltip/jira_ticket_tooltip.tsx

```diff
@@ -0,0 +1,305 @@
+import React, {Fragment, PureComponent} from 'react';
+import './ticketStyle.scss';
+
+import {Instance, GetConnectedResponse} from 'types/model';
+
+export type Props = {
+    href: string;
+    connected: boolean;
+    isloaded?: boolean;
+    assigneeName?: string;
+    assigneeAvatar?: string;
+    labels?: any[];
+    versions?: string;
+    description?: string;
+    summary?: string;
+    ticketId?: string;
+    jiraIcon?: string;
+    statusKey?: string;
+    issueIcon?: string;
+    defaultUserInstanceID?: string;
+    installedInstances: Instance[];
+    connectedInstances: Instance[];
+    getIssueByKey: (ticketId: string, instanceID: string) => void;
+    getConnected: () => Promise<GetConnectedResponse>;
+    setTicket?: (ticketDetails: {}) => void;
+}
+export default class TicketPopover extends React.PureComponent<Props> {
+    truncateString(str: string, num: number) {
+        if (num > str.length) {
+            return str;
+        }
+        return `${str.substring(0, num)}...`;
+    }
+
+    constructor(Props) {
+        super(Props);
+        let ticketID = '';
+        if (this.props.href.includes('selectedIssue')) {
+            ticketID = this.props.href.split('selectedIssue=')[1].split('&')[0];
+        }
+
+        if (ticketID === '' && this.props.href.includes('atlassian.net/browse')) {
```

Comment:
will this trigger on autolink-processed links?

Reply:
yes it does work after autolink being processed , can find a recording of same
Screen Recording 2022-03-02 at 9.39.18 PM